src/shared/containers/SetOrg.tsx

```diff
@@ -311,13 +311,11 @@ const SetOrg: FC = () => {
             />
           )}
           ,
-          {isFlagEnabled('onboardCLI') && (
```

Comment:
This is getting released to OSS as well, correct?

Reply:
Just like other flows, I think so!

Reply:
Looks good!  I don't think this needs any changes then - just wanted to make sure this doesn't need a `CLOUD` check.